multichain-testing/test/ica-channel-close.test.ts

```diff
@@ -280,25 +283,28 @@ const channelCloseInitScenario = test.macro({
     console.log(
       `Initiating channelCloseInit for dst: ${JSON.stringify(dst)} src: ${JSON.stringify(src)}`,
     );
-    t.throws(
-      () => hermes.channelCloseInit(chainName, dst, src),
-      { message: /Command failed/ },
-      'hermes channelCloseInit failed from agoric side for ICA',
-    );
-    t.throws(
-      () => hermes.channelCloseInit(chainName, src, dst),
-      { message: /Command failed/ },
-      `hermes channelCloseInit failed from ${chainName} side for ICA`,
-    );
-
+    try {
+      t.log('relayer channelCloseInit failed from agoric side for ICA');
+      relayer.channelCloseInit(chainName, dst, src);
+    } catch (e) {
+      t.log(e); // hermes relayer throws, but go-relayer does not
+    }
+    try {
+      t.log(`relayer channelCloseInit failed from ${chainName} side for ICA`);
+      relayer.channelCloseInit(chainName, src, dst);
+    } catch (e) {
+      t.log(e); // hermes relayer throws, but go-relayer does not
+    }
+    t.log('Sleeping 10 seconds for potential channel closure to propagate');
+    await sleep(10000);
```

Comment:
how was 10s determined?  please make it more legible:
```suggestion
    await sleep(10 * 1000);
```

Reply:
A bit arbitrary, but aiming for at least 3-4 blocks across the two chains for the relayer txs to propagate. Added a comment and made it more legible.